### ISSUE TITLE
Styling: move count of handsets next to the UIStepper

### ISSUE THREAD

**Issue by ES-Anon** [TechWithTyler/Phonepedia#2]
Currently: a user adjusting the Stepper for handset count + max handset, located on the right side of the view, needs to watch the number change off to the left:

<img width="467" alt="Screenshot 2023-07-27 at 3 59 48 PM" src="https://github.com/TechWithTyler/Phone-Booth/assets/98720615/9d5991c5-f620-4382-a22f-b7af9a0f9eb4">

Suggestion: move the number next to the UI stepper

**Comment by TechWithTyler** [comment]
I don't think that can be easily fixed unless the number was to the right of the stepper by using an HStack.
